atc/metric/emitter/influxdb.go

```diff
@@ -45,7 +45,7 @@ func init() {
 func (config *InfluxDBConfig) Description() string { return "InfluxDB" }
 func (config *InfluxDBConfig) IsConfigured() bool  { return config.URL != "" }
 
-func (config *InfluxDBConfig) NewEmitter() (metric.Emitter, error) {
+func (config *InfluxDBConfig) NewEmitter(map[string]string) (metric.Emitter, error) {
```

Comment:
_?

Reply:
Good catch, I'm surprised there was no compiler errors, but turns out this is perfectly valid syntax and behaves the same! Still makes it harder to read so I fixed it anyways